code/API_definitions/simple-edge-discovery.yaml

```diff
@@ -2,7 +2,7 @@
 openapi: 3.0.3
 info:
  title: Simple Edge Discovery
- version: 1.1.0-alpha.1
+ version: 1.1.0-rc.1
```

Comment:
Here the API version `1.1.0-rc.1` is correct.

Question: the API version 1.1.0-alpha.1 was never released, right? (because that would have been then r2.1, but ok if you have decided to skip the alpha release).

Reply:
Thanks Herbert - correct,  1.1.0-alpha.1 was never released.